src/components/Instructions.component.js

```diff
@@ -5,16 +5,24 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
-    return (
-      <div className="instructions">
-        Follow the instructions on the README to get started!
-        <img
-          className="instructions__logo"
-          src="https://uiuc.hack4impact.org/img/colored-logo.png"
-          alt="h4i logo"
-        />
-      </div>
-    )
+    if (this.props.shouldDisplayImage) {
+      return (
+        <div className="instructions">
+          Follow the instructions on the README to get started!
+          <img
+            className="instructions__logo"
+            src="https://uiuc.hack4impact.org/img/colored-logo.png"
+            alt="h4i logo"
+          />
+        </div>
+      )
+    } else {
+      return (
+        <div className="instructions">
+          Follow the instructions on the README to get started!
+        </div>
+      )
+    }
```

Comment:
this code is pretty wet right now. you can make it more dry by making use of [boolean conditional rendering](https://reactjs.org/docs/conditional-rendering.html)